db/models.py

```diff
@@ -1 +1,40 @@
 from django.db import models
+from django.db.models import DateTimeField
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(
+        Race,
+        related_name="skill_set",
+        on_delete=models.CASCADE
+    )
+
+
+class Guild(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(null=True, blank=True)
+
+
+class Player(models.Model):
+    nickname = models.CharField(max_length=255, unique=True)
+    email = models.EmailField(max_length=255, default="<EMAIL>")
+    bio = models.CharField(max_length=255)
+    race = models.ForeignKey(
+        Race,
+        related_name="players",
+        on_delete=models.CASCADE
+    )
+    guild = models.ForeignKey(
+        Guild,
+        related_name="guild_set",
```

Comment:
The `related_name` for the `guild` ForeignKey in the `Player` model is currently set to `guild_set`. It would be more descriptive to use a name like `members` or `players` to better reflect the relationship.